cosmo_tester/test_suites/manager_of_managers/regional_cluster.py

```diff
@@ -470,7 +473,7 @@ def _populate_status_output(self):
 
     def verify_installation(self):
         super(AbstractRegionalCluster, self).verify_installation()
-        assert self.master_ip
+        assert self.lb_ip
```

Comment:
i think endpoint or cluster_endpoint or endpoint_address or endpoint_ip would be better.

Reply:
Sure, will update it